disperser/cmd/apiserver/main.go

```diff
@@ -161,6 +162,10 @@ func RunDisperserServer(ctx *cli.Context) error {
 	bucketName := config.BlobstoreConfig.BucketName
 	logger.Info("Blob store", "bucket", bucketName)
 	if config.DisperserVersion == V2 {
+		prover, err := prover.NewProver(&config.EncodingConfig, true)
```

Comment:
The prover setup takes around ~20 seconds with 16 cpus since we need to load the G2 points. Also is the disperser still a singleton in V2?

Reply:
I don't think it has to be a singleton anymore now that payments aren't handled locally. @mooselumph is that correct?